actions/commands.py

```diff
@@ -7,53 +7,73 @@
 from .helper import ArgumentParser, translateError
 from .request_wrappers import AuthorizedRequest
 from . import config
-from .errors import error_messages
 
 client = WebClient(token=config.SLACK_BOT_TOKEN)
 
 
 @csrf_exempt
 def create_ticket(request):
     required_fields = ["title"]
-    url = f"{config.API_ROOT}/api/teams/1/tickets/"
-
-    data = request.POST
-    channel_id = data.get("channel_id")
-    text = data.get("text")
-    args = ArgumentParser.parse_args(text)
+    TEAM_ID = 1
+    ticket_url = f"{config.API_ROOT}/api/teams/{TEAM_ID}/tickets/"
+    members_url = f"{config.API_ROOT}/api/teams/{TEAM_ID}/members/"
+
+    slack_data = request.POST
+    channel_id = slack_data.get("channel_id")
+    args = ArgumentParser.parse_args(slack_data.get("text"))
+    user_id = slack_data.get("user_id")
+    api_request = AuthorizedRequest(user_id=user_id)
 
     if not all(field in args for field in required_fields):
         client.chat_postEphemeral(
             channel=channel_id,
-            text=error_messages["required_field"].format("/ticket", "--title"),
-            user=data.get("user_id"),
+            text=f'/ticket syntax error:\n\t/ticket --title "Title" [--asgn @username --desc "Description"]',
```

Comment:
if you're going out of your way to make an errors "module", you should probably include this error message in with it.